packages/concerto-analysis/src/compare-config.ts

```diff
@@ -53,6 +53,8 @@ export const defaultCompareConfig: CompareConfig = {
         'optional-property-added': CompareResult.PATCH,
         'required-property-removed': CompareResult.MAJOR,
         'optional-property-removed': CompareResult.MAJOR,
+        'class-declaration-abstract-to-concrete': CompareResult.MINOR,
+        'class-declaration-concrete-to-abstract': CompareResult.MAJOR, 
```

Comment:
How do we decide if the conversions results in a major/minor bump? Is there some sort of modelling convnetion for this?

Reply:
It's major if existing data can become invalid (i.e. it's not backward-compatible). Otherwise, it's minor or patch. I don't know how they decide between those two. @mttrbrts ?

Reply:
I agree with @DS-AdamMilazzo, this result makes sense to me.